cvxportfolio/tests/test_simulator.py

```diff
@@ -38,7 +38,10 @@
 
 import copy
 import datetime
-import multiprocessing
+try: # pragma: no cover
+    from multiprocessing import cpu_count
+except ImportError:
+    from multiprocess import cpu_count
```

Comment:
I think you got this one backwards 😄

Reply:
Thank you! Yes and in fact on second read I think we can drop `multiprocess` from here.